ph5/clients/ph5tostationxml.py

```diff
@@ -147,6 +147,10 @@ def get_args():
 
     parser.add_argument("--uri", action="store", default="",
                         type=str, metavar="uri")
+
+    parser.add_argument("-E", "--emp_resp", action='store', default=False,
```

Comment:
Adding an --emp_resp option seems like a work around for a deeper level issue. Add this flag may be a good  solution because some ph5 experiments may actually have completely empty response tables but this seems like it needs further investigation. Would it ever make sense for a ph5 experiment to have instruments with no responses? In general all broadband instruments must have a response.